### DISCUSSION TITLE
[Question] I recently did a Login and Launcher with Flet... Flet was updated and now I get this error.

### DISCUSSION THREAD

**Discussion by kevinyablonkis** [flet-dev/flet#3086] [Q&A]
### Question

![imagen](https://github.com/flet-dev/flet/assets/147773213/eb432e23-1d8c-4125-8ed1-bb4266bb6481)

### Error message

![imagen](https://github.com/flet-dev/flet/assets/147773213/17d802cc-09e1-45c0-bb5c-cc1887a1d2dc)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by abdokhalil07** [comment]
plz send code to see what's problem
